Source/WUtils.swift

```diff
@@ -23,4 +23,47 @@ public enum xAlignment {
 }
 public enum yAlignment {
     case Top, Center, Bottom
-}
\ No newline at end of file
+}
+
+public class WUtils {
```

Comment:
I could see them potentially being used in an app that someone can consume. ```WTestUtils``` does not get included when someone installs the pod, which is why I chose to include it in ```WUtils``` instead. Any thoughts?